src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -104,9 +115,9 @@ export class EditConcept extends Component {
     event.preventDefault();
   }
 
-  removeNewName(event, uuid) {
```

Comment:
Why the change from uuid to id?

Reply:
since its uuid is already defined in the scope

Reply:
import uuid from 'uuid/v4';
importing uuid module and it's already define so for a local parameter to a function thought of changing it to id